### ISSUE TITLE
Option to keep display off during an Alarm

### ISSUE THREAD

**Issue by chris-asdf** [gabeg805/NFC-Alarm-Clock#28]
Hi,

Thanks for creating this alarm app, I really like the features of it!

What I'm missing is an option to keep the display off while the alarm is ringing. The alarms worked that way (maybe due to misbehavior) on my old phone on Android 9. So I had to press the power button to enable the display and dismiss/snooze an alarm.
On my new phone (android 14) the display is lit during the Alarm.

Do you see a chance to create an option for keeping the display off and save a bit battery?

Thanks and all the best,
Chris

**Comment by gabeg805** [comment]
Yeah that doesn't seem too hard to add. I'm working on some other features right now, but I'll let you know when it's done.

**Comment by chris-asdf** [comment]
Thanks for looking into it!

**Comment by chris-asdf** [comment]
Hi,

thanks for adding this feature.
It works, but was a bit unexpected since I had to unlock my phone to get to the alarm screen, since it was only visible as a notification.
Is there a chance to change this behavior? So that the Alarm is on Top of the lock screen?

All the best, 
Chris

**Comment by gabeg805** [comment]
Hi @chris-asdf, I have it set to show up on the lock screen, but I think because the screen never turns on, maybe Android isn't respecting that setting.  I can set it so that it turns the screen on, but it doesn't keep the screen on?  I'm not sure if it'll work, I'd have to test it, but I think that's the only other option.  Going that route, the screen might stay on for however long your screen timeout is, so it'll still eat up some battery.

What do you think?

**Comment by chris-asdf** [comment]
Hi Gabriel,

that sounds great!
Thanks for considering it and your time 😃

All the best
Chris

**Comment by gabeg805** [comment]
Hi @chris-asdf, I tested it and unfortunately it doesn't work if I turn on the screen.  

Just for clarity, there's two settings.  One to `turn on the screen`, and one to `keep the screen on`.  If you enable the battery saver option, I had it so it disables both of these.  

I just tested it where `turn on screen = true` and it didn't work the way we expected.  It kept the screen on all the same, so I don't think it's going to work.  You'll have to unlock your phone if you want to use the battery saver option 😭

**Comment by chris-asdf** [comment]
Hi Gabriel,

Don't worry! The way it is now is way better for me than before.

So many thanks again!

All the best, 
Chris
